backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/flightleg/BpkFlightLeg.kt

```diff
@@ -74,7 +77,9 @@ fun BpkFlightLeg(
         Row(modifier = Modifier.fillMaxWidth()) {
             carrierLogoContent?.let {
                 Box(
-                    modifier = Modifier.padding(top = BpkSpacing.Sm),
+                    modifier = Modifier
+                        .padding(top = BpkSpacing.Sm)
```

Comment:
Worth also adding the size as a constraint here

Reply:
I thought about it, too. I'll add it.